core/src/main/java/com/es/core/model/phone/PhoneRowMapper.java

```diff
@@ -0,0 +1,48 @@
+package com.es.core.model.phone;
+
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashSet;
+
+public class PhoneRowMapper implements RowMapper<Phone> {
```

Comment:
You will need also to register this bean in xml config or mark it with annotation